### DISCUSSION TITLE
Configure agent to ignore parent spans?

### DISCUSSION THREAD

**Discussion by stevenzyang** [open-telemetry/opentelemetry-java-instrumentation#6905] [Q&A]
We have a netty service that is only called by clients outside of our system, so we'd like to ignore tracing information coming in from miscellaneous clients. Is it possible to configure the agent to do this?

**Comment by mateuszrzeszutek** [comment]
Hey @stevenzyang ,
This use case is not supported out of the box. However, you should be able to make it work by creating a custom [extension](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/examples/extension) with a custom [`ConfigurablePropagatorProvider`](https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure-spi/src/main/java/io/opentelemetry/sdk/autoconfigure/spi/ConfigurablePropagatorProvider.java) implementation that returns a `TextMapPropagator` that simply does nothing on `extract()`.
